src/components/ArticleFilterProvider.jsx

```diff
@@ -1,10 +1,13 @@
+import { useParams } from "react-router-dom"
 import ArticleList from "./ArticleList"
 
 function ArticleFilterProvider() {
```

Comment:
Seeing as the topic isn't being used by anything else in this component, is the component necessary? Could we just get the topic in the article list component?